checksums.json

```diff
@@ -1,147 +0,0 @@
-{
```

Comment:
If the file is generated every time we build the module then I would prefer it be added to the gitignore.  I use `git add --all` when staging files, and it'd be easy for `checksums.json` to slip through the cracks if I was committing a large number of changes across many files.